tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/StorageManager.java

```diff
@@ -538,7 +538,8 @@ public Appender getAppender(OverridableConf queryContext,
     if (scannerClass == null) {
       scannerClass = conf.getClass(
           String.format("tajo.storage.scanner-handler.%s.class", handlerName), null, Scanner.class);
-      SCANNER_HANDLER_CACHE.put(handlerName, scannerClass);
+      if (scannerClass != null)
```

Comment:
Hi, @blrunner .

I added that because SCANNER_HANDLER_CACHE raises NPE when `getClass` function fails. Even in case that Tajo cannot find a scanner-handler class, I thought it is safe that Tajo shows a reasonable message instead of NPE like the following. 
```
default> select * from orc1; 
ERROR: Unknown Storage Type: ORCFILE
```
This sub-task issue provides ORCFile table creation related patch. This patch supports creating and reading the created table (of course, empty). Until the very next issue supports reading non-empty ORCFile table, 'IF' will show the above message instead of NPE.

Otherwise, it is also possible to throw NonImplementedExcpetion for ORCFile table.

@blrunner , could you give me some advice for more preferred way in Tajo?